RENOVATE.md

```diff
@@ -165,9 +166,40 @@ The final package rule should allow renovate to create new PRs to test out new u
 released, e.g. PHP 8.2.0-rc1. These PRs are not automatically merged, even if all checks are green, and additionally
 they will be assigned the label "Awaiting Maintainer Response".
 
+## Security Updates Only Configuration
+
+Extending the default, the file `renovate-config-security-updates-only.json` is made specifically for use in Laminas
+repositories that have been marked as receiving "security-updates-only".
+
+### Presets
+
+```json
+"extends": [
+   "local>laminas/.github:renovate-config",
+   ":maintainLockFilesDisabled"
+],
+```
+
+- **local>laminas/.github:renovate-config** - As stated, the first preset extends the default configuration described
+above.
+- **[:maintainLockFilesDisabled](https://docs.renovatebot.com/presets-default/#maintainlockfilesdisabled)** - Only
+update lock files when `composer.json` is updated.
+
+### Package Rules
+
+```json
+"packageRules": [
+   {"matchPackagePatterns": ["*"], "enabled": false},
+   {"matchPackageNames": ["php"], "enabled": true}
```

Comment:
The difference between this PR and the last one, is that the "rangeStrategy" will inherit from the default laminas config for PHP. All we need to do really is disable updates for all packages and then reenable them for PHP only.